lib/source-info.coffee

```diff
@@ -31,12 +32,32 @@ module.exports =
     currentLine: ->
       @_currentLine ||= unless @_currentLine
         editor = atom.workspace.getActiveTextEditor()
-        cursor = editor and editor.getCursor()
+        cursor = editor and editor.getLastCursor()
         if cursor
           cursor.getBufferRow() + 1
         else
           null
 
+    minitestRegExp: (text)->
+      @_minitestRegExp ||= unless @_minitestRegExp 
+        value = text.match(/"(.*?)"/) if text?
+        if value
+          value[1]
+        else
+          ""
+
+    isMiniTest: ->
+      i = @currentLine() - 1
+      regExp = new RegExp(/^(\s+)(should|test|it)\s+['""'](.*)['""']\s+do\s*(?:#.*)?$/)
```

Comment:
Also a unit-test style minitest file contains a class that inherits from `Minitest::Test`.